src/utils/allowedHTML.ts

```diff
@@ -11,3 +17,27 @@ const IFRAME_TAG_REGEX = new RegExp("(<iframe.*/iframe>)", "gm")
 export const getLengthWithoutTags = (text: string): number => {
   return text.replace(IFRAME_TAG_REGEX, "").length
 }
+
+/**
+ * Utility method for determinign if the given HTML embed code adheres to our
+ * Content Security Policy.
+ */
+export const isEmbedCodeValid = (
+  csp: Awaited<ReturnType<typeof CspService.get>> | undefined,
+  embedCode: string | undefined
+) => {
+  if (!csp || !embedCode) return false
+
+  const {
+    isCspViolation: checkedIsCspViolation,
+    sanitisedHtml: CSPSanitisedHtml,
+  } = checkCSP(csp, embedCode)
+  DOMPurify.sanitize(CSPSanitisedHtml)
+
+  // Using FORCE_BODY adds a fake <remove></remove>
```

Comment:
what's this forcebody about?

Reply:
This is part of the DOMPurify configuration settings, in which we added a `FORCE_BODY` to support sanitisation of `<script>` tags that appear at the start of the content. This is done in `src/layouts/EditPage/utils.ts`.